lnrpc/autopilotrpc/autopilot_server.go

```diff
@@ -65,6 +65,9 @@ type Server struct {
 	started  int32 // To be used atomically.
 	shutdown int32 // To be used atomically.
 
+	// Required by the grpc-gateway/v2 library for forward compatibility.
+	UnimplementedAutopilotServer
```

Comment:
nit: move to be the first field.

Reply:
Can't do it here as that would break the struct alignment for the atomic variables (which can cause crashes on some architectures). In fact I found another place where this needed to be moved. Added comments to the places where the order is important.

Reply:
TIL this is important :+1:

Reply:
Yeah, I learned this the hard way by producing a panic in an RC with by a pure refactor PR... Now I also know why the `// To be used atomically` comment is important.